packages/commons/src/index.ts

```diff
@@ -5,3 +5,4 @@ export * as ContextExamples from './samples/resources/contexts';
 export * as Events from './samples/resources/events';
 export * from './types/middy';
 export * from './types/utils';
+export * from './middleware';
```

Comment:
This was excluded purposefully. 

The `middleware` folder is specific to Middy-related stuff, which might/might not bring some dependencies.

We don't want to put this in the default export and we want consumers to be explicitly importing the path `@aws-lambda-powertools/commons/lib/middleware` (for now, `@aws-lambda-powertools/commons/middleware` in v2).

Reply:
They are both middleware, but of two different kinds, I would consider creating another subfolder called `user-agent` or something that points to the AWS SDK instead.

Reply:
Good point, I will put it on the top level